tests/test_datasets.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import unittest
+
+import os
+import glob
+
+from neurolib.utils.loadData import Dataset
+import neurolib.utils.functions as func
+
+
+class TestDatasets(unittest.TestCase):
+    def test_dataset_gw(self):
+        ds = Dataset("gw")
+        assert ds.Cmat.shape == (80, 80)
```

Comment:
not needed, but `unittest` provides very nice `self.assertTrue`, `assertEqual`, `assertListEqual`, etc.. better to use those since you will get much more clear messages if the test fails...
here you would use `self.assertTupleEqual`

similarly, all asserts below

Reply:
I can just replace all asserts? neat

Reply:
yep, but it depends on what you are asserting... if general equality you can `self.assertEqual`, but you have a lot of options> https://docs.python.org/3/library/unittest.html#assert-methods

Reply:
also, this is a nice cheat sheet: https://kapeli.com/cheat_sheets/Python_unittest_Assertions.docset/Contents/Resources/Documents/index